docs/redoc/master/openapi.json

```diff
@@ -7299,18 +7299,18 @@
             "minimum": 0,
             "nullable": true
           },
-          "read_rate_limit_per_sec": {
-            "description": "Max number of read operations per second per shard per peer",
+          "read_rate_limit_per_minute": {
+            "description": "Max number of read operations per minute per replica",
             "type": "integer",
             "format": "uint",
-            "minimum": 0,
+            "minimum": 1,
             "nullable": true
           },
-          "write_rate_limit_per_sec": {
-            "description": "Max number of write operations per second per shard per peer",
+          "write_rate_limit_per_minute": {
+            "description": "Max number of write operations per minute per replica",
             "type": "integer",
             "format": "uint",
-            "minimum": 0,
+            "minimum": 1,
             "nullable": true
           }
```

Comment:
I agree with @timvisee here. It might make things easier in the future if we just name it `write_rate_limit`, and specify it is per minute in the description. If we ever want to change the time frame, we can extend with another field or an enum

Reply:
Alright, let's for renaming and adjusting the description :+1:

Reply:
done in https://github.com/qdrant/qdrant/pull/5597/commits/3834182004a1206680ad108c55d5a47e343ea116

Reply:
Thank you! :pray: